traffic_incident_parser/src/traffic_incident_parser_worker.cpp

```diff
@@ -282,14 +282,19 @@ namespace traffic_incident_parser
             RCLCPP_WARN_STREAM(logger_->get_logger(), "Traffic Incident Parser received traffic control message, but it has not loaded the map yet. Returning empty list");
             return {};
         }
+        if (projection_msg_ == "")
+        {
+            RCLCPP_WARN_STREAM(logger_->get_logger(), "Traffic Incident Parser received traffic control message, but georeference has not loaded yet. Returning empty list");
```

Comment:
This should be "... is composing a Traffic Control Message, but ...", would you mind updating it here and in the above warn statement as well?

Reply:
Revised